### ISSUE TITLE
CD: fix ghcr push error

### ISSUE THREAD

**Issue by casperdcl** [premAI-io/prem-services#81]
Docker images are being pushed to the [org packages](https://github.com/orgs/premAI-io/packages) rather than [repo packages](https://github.com/orgs/premAI-io/packages?repo_name=prem-services). However the repo's default CI `GITHUB_TOKEN` [lacks permissions to push to the org](https://github.com/premAI-io/prem-services/actions/runs/5727539409/job/15534055656).

- [ ] create a [machine user (bot) account](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#personal-accounts)
- [ ] create PAT (docs currently say ["classic" rather than "fine grained"](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry) -- worth checking whether "fine grained" actually works though)
- [ ] update CI workflow to use said PAT

**Comment by tiero** [comment]
did you tried adding `packages: write` under `permissions` for the GH Action job?

**Comment by casperdcl** [comment]
yes that's already in `main` https://github.com/premAI-io/prem-services/blob/8e4e2a368909f54d27e32b3fca80a1e2b6637279/.github/workflows/ci.yml#L56-L57 but only scoped to this repo rather than org :/
